docs/utils.js

```diff
@@ -40,3 +40,7 @@ export const fakeIntentCreate = (action, doctype, options) => {
 export const t = (path) => {
   return get(translations, path)
 }
+
+export const isArgos = () => {
+  return navigator.userAgent.includes('Argos')
+}
```

Comment:
It seems that we have a `isTesting` variable to take "open" snapshot. Should we set `isTesting` to true when we are in a jest context but also when the userAgent is Argos? Like that no need to do : 

`menuIsOpen={isArgos() || isTesting}`

Reply:
Correct, I made the change, thanks